das/options.go

```diff
@@ -36,20 +36,24 @@ type Parameters struct {
 	SampleFrom uint64
 
 	// SampleTimeout is a maximum amount time sampling of single block may take until it will be
-	// canceled
+	// canceled. High ConcurrencyLimit value may increase sampling time due to node resources being
+	// divided between parallel workers. SampleTimeout should be adjusted proportional to
+	// ConcurrencyLimit.
 	SampleTimeout time.Duration
 }
 
 // DefaultParameters returns the default configuration values for the daser parameters
 func DefaultParameters() Parameters {
 	// TODO(@derrandz): parameters needs performance testing on real network to define optimal values
 	// (#1261)
+	concurrencyLimit := 16
 	return Parameters{
 		SamplingRange:           100,
-		ConcurrencyLimit:        16,
+		ConcurrencyLimit:        concurrencyLimit,
 		BackgroundStoreInterval: 10 * time.Minute,
 		SampleFrom:              1,
-		SampleTimeout:           time.Minute,
+		// SampleTimeout = block time * max amount of catchup workers
+		SampleTimeout: 15 * time.Second * time.Duration(concurrencyLimit),
```

Comment:
We should pass this block time from as a param tbh. There are multiple places where we rely on it, and changing it in one place is convenient. modp2p.Blocktime

Reply:
We would also need to merge https://github.com/celestiaorg/celestia-node/pull/1773

Reply:
No, it can just expect the caller to pass it in DefaultParams

Reply:
But they are not really default then

Reply:
We should then just reassign those on the nodebuilder lever

Reply:
We can do it too. But we need to set default and it should not depend on node pkg. Maybe magic number is not that bad, considering we don't really care about being exact here, just using the assumption to make some pre-defined default.